modules/services/crashlogging/src/main/java/au/com/shiftyjelly/pocketcasts/crashlogging/PocketCastsCrashLoggingDataProvider.kt

```diff
@@ -37,22 +36,18 @@ class PocketCastsCrashLoggingDataProvider @Inject constructor(
 
     override val sentryDSN: String = BuildConfig.SENTRY_DSN
 
-    override val user: Flow<CrashLoggingUser?> = settings.linkCrashReportsToUser.flow.map { linkCrashReportsToUser ->
-        if (linkCrashReportsToUser) {
-            syncManager.getEmail()?.let { email ->
-                CrashLoggingUser(
-                    email = email,
-                    userID = "",
-                    username = "",
-                )
-            }
-        } else {
-            null
+    override val user: Flow<CrashLoggingUser?> = observeUser.invoke().map { userMail ->
+        userMail?.let {
+            CrashLoggingUser(
+                email = it.email,
+                userID = "",
+                username = "",
+            )
```

Comment:
This will be better - empty strings are not very useful, it'll only clutter Sentry dashboard. https://github.com/Automattic/Automattic-Tracks-Android/pull/218